.github/workflows/cpu-packages.yml

```diff
@@ -10,20 +10,15 @@ on:
 jobs:
   build-conda:
     name: Build Conda
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        python-version: [3.8]
```

Comment:
Wondering if we could add 3.9 and 3.10 here so the builds run in separate actions?